packages/sol-cov/src/coverage_manager.ts

```diff
@@ -154,7 +160,10 @@ export class CoverageManager {
                 }
             } else {
                 // Contract creation transaction
-                const bytecode = (traceInfo as TraceInfoNewContract).bytecode;
+                let bytecode = (traceInfo as TraceInfoNewContract).bytecode;
+                if (bytecode.startsWith('0x')) {
+                    bytecode = bytecode.slice(2);
+                }
```

Comment:
Since we repeat this logic twice, let's more to a util method: `utils.removeHexPrefix()` or use one in `ethereumjs_utils`